README.md

```diff
@@ -1,135 +1,96 @@
+### Kurtosis Devnet Tests
 
-`op-geth` embeds upstream geth’s version inside its own version as follows: `vMAJOR.GETH_MAJOR GETH_MINOR GETH_PATCH.PATCH`.
-Basically, geth’s version is our minor version.
-For example if geth is at `v1.12.0`, the corresponding op-geth version would be `v1.101200.0`.
-Note that we pad out to three characters for the geth minor version and two characters for the geth patch version.
-Since we cannot left-pad with zeroes, the geth major version is not padded.
+For full e2e tests we leverage optimism's [kurtosis devnet](./kurtosis-devnet/README.md). See the config file to spin up a devnet with eigenda-proxy in [memstore](./kurtosis-devnet/eigenda-memstore.yaml) mode and [holesky](./kurtosis-devnet/eigenda-holesky.yaml) mode, as well as the available eigenda group commands in the [justfile](./kurtosis-devnet/justfile):
+```sh
+$ just --list
+Available recipes:
+    ...
 
-See the [Node Software Releases](https://docs.optimism.io/builders/node-operators/releases) page of the documentation for more information about releases for the latest node components.
+    [eigenda]
+    eigenda-holesky-devnet-clean
+    eigenda-holesky-devnet-start                     # EigenDA devnet that uses eigenda-proxy connected to eigenda holesky testnet network
+    eigenda-memstore-devnet-clean
+    eigenda-memstore-devnet-configs
+    eigenda-memstore-devnet-failback
+    eigenda-memstore-devnet-failover                 # to failover to ethDA. Use `eigenda-memstore-devnet-failback` to revert.
+    eigenda-memstore-devnet-grafana
+    eigenda-memstore-devnet-restart-batcher          # Restart batcher with new flags or image.
+    eigenda-memstore-devnet-start                    # EigenDA devnet that uses the eigenda-proxy in memstore mode (simulates an eigenda network but generates random certs)
+    eigenda-memstore-devnet-sync-status
+    eigenda-memstore-devnet-test
+```
 
-The full set of components that have releases are:
+## Releases and Branching Strategy
 
-- `op-batcher`
-- `op-contracts`
-- `op-challenger`
-- `op-node`
-- `op-proposer`
+We maintain an `eigenda-develop` branch which is our main development branch which keeps a linear history containing new feature work, fixes, as well as upstream merges. To make downstream integrations easier, we create release-specific branches which contain cleaned up history of commits on top of a specific upstream release. For example, the second eigenda-fork release in the below picture would be named `op-batcher/v1.11.2-eigenda.2`, and will consist of a cleaned-up history of commits (one per feature/service pair) on top of the upstream [op-batcher/v1.11.2](https://github.com/ethereum-optimism/optimism/releases/tag/op-batcher%2Fv1.11.2) release. We will strive to make our releases on top of op [production releases](https://github.com/ethereum-optimism/optimism?tab=readme-ov-file#production-releases), unless an urgent fix is needed.
```

Comment:
> We maintain an `eigenda-develop` branch which is our main development branch which keeps a linear history containing new feature work, fixes, as well as upstream merges.

Run-on sentence, consider the following potential remedy:

"Our main development branch, `eigenda-develop`, contains a linear history with new feature work and fixes, as well as upstream merges."

Reply:
[3a581a7](https://github.com/Layr-Labs/optimism/pull/37/commits/3a581a75dd2f0ee6f26de854278d04f36a1a2662)